docs/source/gateway/index.rst

```diff
@@ -240,19 +240,19 @@ With the rapid development of LLMs, there is no guarantee that this list will be
 below can be used as a helpful guide when configuring a given route for any newly released model types as they become available with a given provider.
 ``N/A`` means that the provider currently doesn't support the route type.
 
-+--------------------+--------------------------+--------------------+------------------+-----------------------------+--------------------------+--------------------------+
-| Route Type         | OpenAI                   | MosaicML           | Anthropic        | Cohere                      | Azure OpenAI             | MLflow                   |
-+====================+==========================+====================+==================+=============================+==========================+==========================+
-| llm/v1/completions | - gpt-3.5-turbo          | - mpt-7b-instruct  | - claude-1       | - command                   | - text-davinci-003       | - MLflow served models*  |
-|                    | - gpt-4                  | - mpt-30b-instruct | - claude-1.3-100k| - command-light-nightly     | - gpt-35-turbo           |                          |
-|                    |                          | - llama2-70b-chat† |                  |                             |                          |                          |
-+--------------------+--------------------------+--------------------+------------------+-----------------------------+--------------------------+--------------------------+
-| llm/v1/chat        | - gpt-3.5-turbo          | - llama2-70b-chat† | N/A              | N/A                         | - gpt-35-turbo           | - MLflow served models*  |
-|                    | - gpt-4                  |                    |                  |                             | - gpt-4                  |                          |
-+--------------------+--------------------------+--------------------+------------------+-----------------------------+--------------------------+--------------------------+
-| llm/v1/embeddings  | - text-embedding-ada-002 | - instructor-large | N/A              | - embed-english-v2.0        | - text-embedding-ada-002 | - MLflow served models** |
-|                    |                          | - instructor-xl    |                  | - embed-multilingual-v2.0   |                          |                          |
-+--------------------+--------------------------+--------------------+------------------+-----------------------------+--------------------------+--------------------------+
++--------------------+--------------------------+--------------------+------------------+-----------------------------+--------------------------+--------------------------+--------------------------+
+| Route Type         | OpenAI                   | MosaicML           | Anthropic        | Cohere                      | Azure OpenAI             | MLflow                   | AWS Bedrock              |
```

Comment:
Thanks so much for adding this to docs! Can we add an example in https://github.com/mlflow/mlflow/tree/master/examples/gateway as well? This is awesome!